src/request/derive-address/index.html

```diff
@@ -67,13 +67,13 @@
 
```

Comment:
Please remove all changes of this file, as this is done in #191 and will lead to merge conflicts.